src/color_picker_dropdown.coffee

```diff
@@ -0,0 +1,106 @@
+###*
+ * ColorPickerDropdownComponent
+ *
+ * This is an internal-use component to control the logic of the dropdown of
+ * the color picker. It contains the color palette and the custom color input
+ * @class
+ * @augments {Ember.Component, Ember.Widgets.BodyEventListener,
+ *            Ember.Widgets.ColorPickerMixin}
+###
+Ember.Widgets.ColorPickerDropdownComponent =
+Ember.Component.extend Ember.Widgets.BodyEventListener,
+Ember.Widgets.ColorPickerMixin,
+  layoutName: 'color-picker-dropdown'
+  dropdownClass: null
+
+  selectedColor: ''
+
+  customColor: ''
+
+  colorRows:
+    Ember.A([
+      Ember.A([
+        '#000000',
+        '#111111',
+        '#434343',
+        '#666666',
+        '#999999',
+        '#AAAAAA',
+        '#B7B7B7',
+        '#CCCCCC',
+        '#D9D9D9',
+        '#EFEFEF',
+        '#F3F3F3',
+        '#FFFFFF'
+      ])
+      Ember.A([
+        '#001F3F',
+        '#0074D9',
+        '#7FDBFF',
+        '#39CCCC',
+        '#2ECC40',
+        '#01FF70',
+        '#FFDC00',
+        '#FF851B',
+        '#FF4136',
+        '#85144B',
+        '#B10DC9',
+        'transparent'
+      ])
+    ])
+
+  isColorTransparent: Ember.computed.equal 'selectedColorRGB', 'transparent'
+
+  setCustomColor: Ember.on 'init', Ember.observer ->
+    selectedColor = @get 'selectedColor'
+    selectedColor = @colorToHex(selectedColor)
+    return @set('customColor', '') if @get('colorRows').find (row) ->
```

Comment:
This was there before. I agree that it's weird here, but I would prefer to spend time removing this entire observer separately.